### PR TITLE
Load cluster info from configmap only if iks is enabled

### PR DESCRIPTION
**Pull request by meghnasiingh**
Signed-off-by: Meghna Singh <Meghna.Singh1@ibm.com>

### CODE REVIEW

File: pkg/ibmcloudprovider/volume_provider.go [hunk 1]
```diff
@@ -61,15 +62,16 @@ func NewIBMCloudStorageProvider(configPath string, logger *zap.Logger) (*IBMClou
 		conf.VPC.APIVersion = "2020-07-02" // setting default values
 	}
 
+	var clusterInfo *utils.ClusterInfo
```
Comment (arahamad) [inline pkg/ibmcloudprovider/volume_provider.go:65]: can we make clusterInfo var inside the if condition

File: pkg/ibmcloudprovider/volume_provider.go [hunk 2]
```diff
@@ -61,15 +62,16 @@ func NewIBMCloudStorageProvider(configPath string, logger *zap.Logger) (*IBMClou
...
 	logger.Info("Fetching clusterInfo")
-	clusterInfo, err := utils.NewClusterInfo(logger)
-	if err != nil {
-		logger.Fatal("Unable to load ClusterInfo", local.ZapError(err))
-		return nil, err
-	}
-	logger.Info("Fetched clusterInfo..")
-	if conf.Bluemix.Encryption || conf.VPC.Encryption {
-		if os.Getenv("IKS_ENABLED") == "True" {
+	if conf.IKS != nil && conf.IKS.Enabled || strings.Contains(os.Getenv("IKS_ENABLED"), "True") {
```
Comment (arahamad) [inline pkg/ibmcloudprovider/volume_provider.go:67]: can we avoid `Contains` method may be we can use equality operator

**Review by arahamad — CHANGES_REQUESTED** [review]
can you please check if you can do these changes

File: pkg/ibmcloudprovider/volume_provider.go (hunk 1 above)
Comment (meghnasiingh) [reply pkg/ibmcloudprovider/volume_provider.go:65]: We are using this variable outside this block also. So, had to define outside block

**Review by prankulmahajan — APPROVED** [review]
LGTM

**Review by arahamad — APPROVED** [review]
LGTM

**Pull request merged**